### PR TITLE
DOCS 8287: Duplicate a project in APID

### PR DESCRIPTION
**Pull request by rddheath**

### CODE REVIEW

File: modules/ROOT/pages/design-duplicate-project.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+. Open the project.
+. Click the gear in the top-right corner of the editor.
+. Select *Duplicate*.
+. Name the duplicate project and click *Duplicate*.
+
+== Result
+
+API Designer opens a new browser tab that displays the new project. If the original project is in the text editor, the duplicate project is opened in the text editor. If the original project is in the visual editor, the duplicate project is opened in the visual editor.
+
+If the original was in a sub-branch, the duplicate is on the master branch.
```
Comment (marcoskhabie) [inline modules/ROOT/pages/design-duplicate-project.adoc:27]: If the project was in a sub-branch the duplicate is on that branch, not on master.

File: modules/ROOT/pages/design-duplicate-project.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
+= Duplicate a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+You can duplicate projects in API Designer. You might want to duplicate a project if you need to make extensive changes to the project without losing the original on which the duplicate project is based.
```
Comment (kbostrom) [inline modules/ROOT/pages/design-duplicate-project.adoc:7]: Strictly subjective:
```suggestion
You can duplicate projects in API Designer: for example, if you need to make extensive changes to the project without losing the original on which the duplicate project is based.
```

File: modules/ROOT/pages/design-duplicate-project.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+== Before You Begin
+
+* Ensure that the mocking service is not turned on.
+* Ensure that the project is not in read-only mode.
+
+== Procedure
+
+. Open the project.
+. Click the gear in the top-right corner of the editor.
```
Comment (kbostrom) [inline modules/ROOT/pages/design-duplicate-project.adoc:19]:
```suggestion
. Click the gear icon in the top-right corner of the editor.
```

File: modules/ROOT/pages/design-duplicate-project.adoc [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+. Select *Duplicate*.
+. Name the duplicate project and click *Duplicate*.
+
+== Result
+
+API Designer opens a new browser tab that displays the new project. If the original project is in the text editor, the duplicate project is opened in the text editor. If the original project is in the visual editor, the duplicate project is opened in the visual editor.
+
+If the original was in a sub-branch, the duplicate is on the same sub-branch.
```
Comment (kbostrom) [inline modules/ROOT/pages/design-duplicate-project.adoc:27]:
```suggestion
If the original was in a subbranch, the duplicate is on the same subbranch.
```
Our style is to not hyphenate after most prefixes; so is Salesforce's.

**Pull request merged**